app/src/main/res/layout-land/admin_auth_activity.xml

```diff
@@ -80,6 +81,7 @@
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginTop="12dp"
+            android:textDirection="locale"
```

Comment:
You can remove this.
The reason why we need this is because of `Force RTL` from developer options.
But instead we should check the app on `pseudoLocale` something like Arabic.
To test this correctly merge your branch with latest `develop` and follow steps mentioned in this https://github.com/oppia/oppia-android/wiki/RTL-Guidelines and based on the result update your xml files.

In general you can remove `textDirection="locale"` and test app for `cibarA [XB]` language and fix UI as per your requirements.

Reply:
done